services/QuillLMS/app/services/unit_template_pseudo_serializer.rb

```diff
@@ -129,6 +133,42 @@ def activities
     activity_hashes.uniq { |a| a[:id] }
   end
 
+  def student_counts_for_previously_assigned_activity(unit, classrooms)
+    classrooms.map do |classroom|
+      classroom_unit = unit.classroom_units.where(classroom_id: classroom[:id]).first
```

Comment:
`.where... .first` can just be `.find_by`.

Reply:
There's actually a [subtle](https://stackoverflow.com/a/56594507) difference between `find_by` and `where(...).first`.

Reply:
That's good to flag. In this instance, where there is only going to be one classroom unit per intersection of classroom and unit, it shouldn't matter, but in other cases it might.